virt/nodes/virt-triggering-vm-failover-resolving-failed-node.adoc

```diff
@@ -6,15 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-If a node fails and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[machine health checks] are not deployed on your cluster, virtual machines (VMs) with `runStrategy: Always` configured are not automatically relocated to healthy nodes. To trigger VM failover, you must manually delete the `Node` object.
-
-[NOTE]
-====
-If you installed your cluster by using xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[installer-provisioned infrastructure] and you properly configured machine health checks, the following events occur:
-
-* Failed nodes are automatically recycled.
-* Virtual machines with xref:../../virt/nodes/virt-node-maintenance.adoc#run-strategies[`runStrategy`] set to `Always` or `RerunOnFailure` are automatically scheduled on healthy nodes.
-====
+If a node fails and link:https://access.redhat.com/articles/7057929[node health checks] are not deployed on your cluster, virtual machines (VMs) with `runStrategy: Always` configured are not automatically relocated to healthy nodes. You must manually trigger the VM failover.
```

Comment:
I am unsure how we should trigger or what steps we plan to add as part of manually trigger VM Failover.